src/structures/game_session.ts

```diff
@@ -492,6 +518,49 @@ export default class GameSession {
         }
 
         this.playSong(guildPreference, messageContext);
+
+        if ([GameType.MC_EASY, GameType.MC_MEDIUM, GameType.MC_HARD].includes(this.gameType)) {
+            const correctChoice = guildPreference.getGuessModeType() === GuessModeType.ARTIST ? this.gameRound.artistName : this.gameRound.songName;
+            const wrongChoices = await getThreeChoices(this.gameType,
+                guildPreference.getGuessModeType(),
+                randomSong.members,
+                correctChoice,
+                randomSong.artistID);
+
+            this.gameRound.correctAnswerUUID = uuid.v4();
+            this.gameRound.incorrectAnswerUUIDs = [
+                uuid.v4(),
+                uuid.v4(),
+                uuid.v4(),
+            ];
+
+            const backupWrongChoices = await getThreeChoices(GameType.MC_EASY,
+                guildPreference.getGuessModeType(),
+                randomSong.members,
+                correctChoice,
+                randomSong.artistID);
+
+            const components: Array<Eris.InteractionButton> = [{ type: 2, style: 1, label: correctChoice, custom_id: this.gameRound.correctAnswerUUID }];
+            for (const [i, choice] of wrongChoices.entries()) {
+                if (choice) {
+                    components.push({ type: 2, style: 1, label: wrongChoices[i], custom_id: this.gameRound.incorrectAnswerUUIDs[i] });
+                } else {
+                    // Fill any null choices with easy choices
+                    components.push({ type: 2, style: 1, label: backupWrongChoices[i], custom_id: this.gameRound.incorrectAnswerUUIDs[i] });
+                }
+            }
+
+            sendInfoMessage(new MessageContext(this.textChannelID), {
+                title: "Choose your guess!",
+                components: [
+                    {
+                        type: 1,
+                        components: _.shuffle(components),
```

Comment:
Do we need to shuffle?

Reply:
Yes, otherwise the correct answer will always be the first component